examples-gallery/plot_parameter_estimation.py

```diff
@@ -0,0 +1,205 @@
+# %%
+"""
+
+Parameter Estimation
+====================
+
+Four noisy measurements of the location of a simple system consisting of a mass
+connected to a fixed point by a spring and a damper. The movement is in a
+horizontal direction. The the spring constant and the damping coefficient
+are to be estimated.
+
+The idea is to set up four sets of eoms, one for each of the measurements, with
+identical parameters, and let opty estimate the parameters.
+
+**State Variables**
+
+- :math:`x_1`: position of the mass of the first system [m]
+- :math:`x_2`: position of the mass of the second system [m]
+- :math:`x_3`: position of the mass of the third system [m]
+- :math:`x_4`: position of the mass of the fourth system [m]
```

Comment:
These are really all the same system. I would call $x_1$ the state variable associated with the first measurement trial.